src/PolynomialSolutions.jl

```diff
@@ -454,6 +512,114 @@ function solve_laplace(Q::Polynomial{N,T}) where {N,T}
     return P
 end
 
+"""
+    solve_anisotropic_laplace(A::AbstractMatrix{T}, Q::Polynomial)
+
+Return a polynomial `P` satisfying the anisotropic Laplace equation `∇ ⋅ (A ∇P) = Q`, `A` an invertible
+matrix. `Q` is required to be homogeneous. Inverse is anisotropic_laplacian.``
```

Comment:
Do you want to cross reference `anisotropic_laplacian`? In any case, there is a dangling ``